build.dev.js

```diff
@@ -0,0 +1,430 @@
+"use strict";
```

Comment:
This file plus build.lib.js and build.prod.js seem to have quite a bit of the same code. Like `log` and `warn` and others. Maybe you can create a build.utils.js and just require the utils into these 3 files. It would make it a lot easier to read through the build for the important pieces.